CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - Support for reporting the stack trace in case of error events, if provided by the player
+- New `ConvivaAnalyticsIntegration.setAutoEndSession` to not end Conviva session automatically from certain player events if set to false (true by default).
```

Comment:
Please also add an entry for the new `reportPlaybackStalled` API.

Reply:
I have added an entry for reportPlaybackStalled in CHANGELOG.md